docs/User Manual.rst

```diff
@@ -742,7 +742,9 @@ from ACE files.
   resonance probability tables treatment
 * DBRC (*optional*, default = no DBRC): list of ZAIDs of nuclides for which DBRC has
   to be applied.
-
+* majorant (*optional*, default = 0): 1 for true; 0 for false; flag to activate the 
```

Comment:
As above, I think this should default to true. I suppose it comes down to which would we prefer to suffer in a worst case - a slower calculation or more memory. I think the memory is more tolerable.